osm2gtfs.py

```diff
@@ -298,5 +293,34 @@ def load_config(file):
 
     return config;
 
+def load_agency(config):
```

Comment:
Just a German style nitpick for the future ;) PEP8 requires two blank lines between methods.